.github/workflows/build.yml

```diff
@@ -22,13 +22,22 @@ jobs:
             matrix:
                 php: [ "8.0", "7.4" ]
                 symfony: [ "^4.4", "^5.2" ]
-                sylius: [ "~1.9.0", "~1.10.0@alpha" ]
+                sylius: ["~1.9.0", "~1.10.0", "~1.11.0"]
                 node: [ ">=12.13.0" ]
                 mysql: [ "8.0" ]
 
                 exclude:
-                    -   php: "8.0"
-                        sylius: "~1.9.0"
+                    - php: "8.0"
+                      sylius: "~1.9.0"
+
+                    - sylius: "~1.11.0"
+                      php: "7.4"
+
+                    - sylius: ~1.10.0
+                      symfony: 4.4
+
+                    - sylius: ~1.11.0
```

Comment:
I think if you already excluded PHP 7.4 in sylius 1.11 then it is unnecessary because Symfony 4.4 require 7.4. But if I am wrong you can just group this exclude rules like : 

- sylius "~1.11.0"
- PHP: "7.4"
- Symfony: "4.4" 

as 1 exclude rule.